app/controllers/users/sessions_controller.rb

```diff
@@ -56,6 +56,14 @@ def timeout
       redirect_to root_url(request_id: request_id)
     end
 
+    def bounced
```

Comment:
We may wanna move this to something like `SpHandoffBounced` controller to separate the logic from sessions

Reply:
Also, we may want an analytics event here